js/views/PageNav.js

```diff
@@ -0,0 +1,67 @@
+import electron from 'electron';
+import { View } from 'backbone';
+import loadTemplate from '../utils/loadTemplate';
+import app from '../app';
+
+const remote = electron.remote;
+
+export default class PageNav extends View {
+  constructor(options) {
+    super({
+      className: 'pageNav',
+      events: {
+        'click .js-navClose': 'navCloseClick',
+        'click .js-navMin': 'navMinClick',
+        'click .js-navMax': 'navMaxClick',
+      },
+      ...options,
+    });
+
+    this.listenTo(app.localSettings, 'change:mac_style_win_controls',
+      this.onWinControlsStyleChange);
+    this.setWinControlsStyle(app.localSettings.get('mac_style_win_controls') ? 'mac' : 'win');
+  }
+
+  setWinControlsStyle(style) {
+    if (style !== 'mac' && style !== 'win') {
+      throw new Error('Style must be \'mac\' or \'win\'.');
+    }
+
+    this.$el.removeClass('winStyleWindowControls macStyleWindowControls');
+    this.$el.addClass(style === 'mac' ? 'macStyleWindowControls' : 'winStyleWindowControls');
+  }
+
+  onWinControlsStyleChange(model, useMacStyle) {
+    this.setWinControlsStyle(useMacStyle ? 'mac' : 'win');
+  }
+
+  navCloseClick() {
+    if (remote.process.platform !== 'darwin') {
+      remote.getCurrentWindow().close();
+    } else {
+      remote.getCurrentWindow().hide();
+    }
+  }
+
+  navMinClick() {
+    remote.getCurrentWindow().minimize();
+  }
+
+  navMaxClick() {
+    if (remote.getCurrentWindow().isMaximized()) {
+      remote.getCurrentWindow().unmaximize();
+      // this.$('.js-navMax').attr('data-tooltip', window.polyglot.t('Maximize'));
+    } else {
+      remote.getCurrentWindow().maximize();
+      // this.$('.js-navMax').attr('data-tooltip', window.polyglot.t('Restore'));
+    }
+  }
+
+  render() {
+    loadTemplate('pageNav.html', (t) => {
```

Comment:
I also slightly changed the loadTemplate API in regards to the file path we pass in. Whereas before, we passed in the full path of the template relative to the app root (e.g. `./js/templates/pageNav.html`), now, by default, loadTemplate will assume that the root is the template folder, so the path you would pass in to reference the template from the example above would be simply `pageNav.html`.

You can pass in an optional root argument if you want a different root path (but, I don't think anywhere in v1 we referenced a template outside of the `templates` folder).